lib/bolt/bolt_option_parser.rb

```diff
@@ -627,7 +627,7 @@ def initialize(options)
              "'success' nodes that succeeded in the last run.") do |rerun|
         @options[:rerun] = rerun
       end
-      define('--noop', 'Execute a task that supports it in noop mode') do |_|
+      define('--noop', 'Execute a task that supports noop or bolt apply in noop mode') do |_|
```

Comment:
This is an unwieldy sentence. It could be read "a task that supports bolt apply", and since we only show this flag for `bolt task run` and `bolt apply` anyway, mentioning `bolt apply` in the text feels redundant. It also isn't helpful if you don't already know what `noop` means. It might be better to say something generic like "Simulate the result without making any changes".

Reply:
I ended up using a modified version of the `puppet apply` noop help text